files/ru/web/javascript/reference/statements/import/index.md

```diff
@@ -138,40 +132,40 @@ import myDefault, { foo, bar } from "/modules/my-module.js";
 
 Такой код выведет ошибку:
 
-##### my-module.js
+- my-module.js:
```

Comment:
this part has been removed from en-US, but for now, I've used unordered list instead